### PR TITLE
OE0-48: fixed imports in tests, fixed test in Patient

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://openimis.atlassian.net/browse/OE0-48

Note: The tests for another resources than Patient will be updated/added as a part of another tickets from E0 (tickets regarding satisfying openIMIS profile). In another resources I only fixed imports etc so as to import resources from new fhir.resource library that I connected in https://openimis.atlassian.net/browse/OE0-18.

### CODE REVIEW

File: api_fhir_r4/tests/mixin/fhirApiUpdateTestMixin.py [hunk 1]
```diff
@@ -33,12 +33,13 @@ def test_put_should_update_correctly(self):
         # create
         self.create_dependencies()
         response = self.client.post(self.base_url, data=self._test_request_data, format='json')
+        response_json = response.json()
```
Comment (delcroip) [inline api_fhir_r4/tests/mixin/fhirApiUpdateTestMixin.py:36]: is it used somewhere ?

Comment (sniedzielski) [reply api_fhir_r4/tests/mixin/fhirApiUpdateTestMixin.py:36]: No, I will remove this. Let me do it.

Comment (sniedzielski) [reply api_fhir_r4/tests/mixin/fhirApiUpdateTestMixin.py]: @delcroip I've removed this.

**Comment by sniedzielski** [comment]
@delcroip You can re-check this PR.

**Review by delcroip — APPROVED** [review]

**Pull request merged**